src/RollbarJsHelper.php

```diff
@@ -0,0 +1,185 @@
+<?php namespace Rollbar;
+
+class RollbarJsHelper
+{
+
+    protected $config;
+    
+    /**
+     * @param array $config The array passed to he should have the same
+     * available options that you can find in Rollbar.js, using symbols or
+     * strings for the keys.
+     */
+    public function __construct($config)
+    {
+        $this->config = $config;
+    }
+    
+    /**
+     * Shortcut method for building the RollbarJS Javascript
+     *
+     * @param array $config @see addJs()
+     * @param string $nonce @see addJs()
+     *
+     * @return string
+     */
+    public static function buildJs($config, $headers = null, $nonce = null)
+    {
+        $helper = new self($config);
+        return $helper->addJs($headers, $nonce);
+    }
+    
+    /**
+     * Build Javascript required to include RollbarJS on
+     * an HTML page
+     *
+     * @param array $headers Response headers usually retrieved through
+     * headers_list() used to verify if nonce should be added to script
+     * tags based on Content-Security-Policy
+     * @param string $nonce Content-Security-Policy nonce string if exists
+     *
+     * @return string
+     */
+    public function addJs($headers = null, $nonce = null)
+    {
+        return
```

Comment:
So this returns one string with two script tags, I think some browsers can execute script tags in arbitrary order so we suggest in the rollbar.js docs putting both the _rollbarConfig variable definition and the snippet inside one script tag. For most browsers it doesn't matter as things are executed top to bottom, but to be safe we should generate one script tag which has both the config var and the snippet.